pages/id/docs/index.mdx

```diff
@@ -9,9 +9,9 @@ labels:
 
 Ada beberapa jenis dokumentasi yang tersedia di situs ini:
 
-- Dokumentasi Referensi API
-- Fitur ES6
-- Panduan
+-   Dokumentasi Referensi API
+-   Fitur ES6
+-   Panduan
```

Comment:
```suggestion
- Dokumentasi Referensi API
- Fitur ES6
- Panduan
```

Reply:
The index.mdx file (ID) has been fixed by removing the spaces. However, when it's translated using crowdin platform, they will always ask for a Pull Request with the format they have, even though it might be different from the previously fixed format. :)

Reply:
If the file is manually fixed over here, Crowdin will not override the file, sorry you're wrong @Mr7Gabriel 😅